ocs_ci/ocs/node.py

```diff
@@ -26,13 +26,17 @@ def get_node_objs(node_names=None):
     nodes_obj = OCP(kind='node')
     node_dicts = nodes_obj.get()['items']
     if not node_names:
-        return [OCS(**node_obj) for node_obj in node_dicts]
+        nodes = [OCS(**node_obj) for node_obj in node_dicts]
+        assert nodes, "Failed to get the nodes OCS objects"
+        return nodes
     else:
-        return [
+        nodes = [
             OCS(**node_obj) for node_obj in node_dicts if (
                 node_obj.get('metadata').get('name') in node_names
             )
         ]
+        assert nodes, "Failed to get the nodes OCS objects"
+        return nodes
```

Comment:
These lines can be moved outside if-else and then L30 & L31 can be removed.

Reply:
good catch